packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -31,11 +30,6 @@ contract SuperchainConfig is Initializable, ISemver {
         REMOVE_SEQUENCER
     }
 
-    /// @notice The address of the systemOwner who may trigger an upgrade or change to critical config values.
-    ///         This will be a DelayedVetoable contract.
-    ///         It can only be modified by an upgrade.
-    bytes32 public constant SYSTEM_OWNER_SLOT = bytes32(uint256(keccak256("superchainConfig.systemowner")) - 1);
```

Comment:
Hmm I don't love leaking the proxy admin abstraction into the implementation here

Reply:
I 100% get where you're coming from with that, it felt a bit dirty to me too. I'm kind of on the fence, but in the interest of finding the best solution, here's my steel man argument for this approach:

The requirement is that both [upgrades](https://www.notion.so/oplabs/External-Security-Council-Product-Requirements-FAQ-10425231ed5a47138c388b8705c313f1?pvs=4#aebe5405146f48dd893ec4b344cb5401), and the [removal of a sequencer](https://www.notion.so/oplabs/External-Security-Council-Product-Requirements-FAQ-10425231ed5a47138c388b8705c313f1?pvs=4#98a95e1bb0b243d980d3b4edb4fbe9f2) must be  enacted by Optimism Governance, and subject to a delay.

The implementation can satisfy that requirement directly by reading the same address from storage, the alternative is to have:

- 2 separate values in the config which we need to assert are identical
- 2 separate storage values that need to be kept in sync

So yes, in one sense the proxy abstraction is leaking into the implementation. But in another sense, the intended behavior is being laid out clearly.

Reply:
I'm going to make sure this is documented in the PR description itself. Resolving.